sympy/calculus/util.py

```diff
@@ -745,10 +746,13 @@ def is_convex(f, *syms, domain=S.Reals):
     .. [5] https://en.wikipedia.org/wiki/Concave_function
 
     """
-
-    if len(syms) > 1:
-        raise NotImplementedError(
-            "The check for the convexity of multivariate functions is not implemented yet.")
+    if type(*syms) != Symbol :
```

Comment:
What does `type(*syms)` do if there is more than one symbol?

Reply:
I'm going to change that according to [this discussion](https://github.com/sympy/sympy/discussions/26485), it's a mistake from mine due to my error about `hessian`